tests/utils/param.py

```diff
@@ -0,0 +1,23 @@
+""" Helper functions for inspecting hyperparameters """
+
+from fastai.basics import *
+
+def get_lrsmoms_paired(learn):
```

Comment:
Not sure those changes are necessary: again why only test the lrs/moms at the end of each epoch only?